inc/saibuffer.h

```diff
@@ -546,6 +546,29 @@ typedef enum _sai_buffer_profile_threshold_mode_t
 
 } sai_buffer_profile_threshold_mode_t;
 
+/**
+ * @brief Enum defining queue actions in case the packet fails to pass the admission control.
+ */
+typedef enum _sai_buffer_profile_packet_admission_fail_action_t
+{
+    /**
+     * @brief Drop the packet.
+     *
+     * Default action. Packet has nowhere to go
+     * and will be dropped.
+     */
+    SAI_BUFFER_PROFILE_PACKET_ADMISSION_FAIL_ACTION_DROP = 0x00000000,
```

Comment:
since this is just enum you don't need to explicitly assign any values, you can remove them to make code more clean